### PR TITLE
Added a Tech Preview note to Helm docs

### PR DESCRIPTION
**Pull request by yhontyk**
- branch 4.3

### CODE REVIEW

File: cli_reference/helm-cli/getting-started-with-helm-on-openshift-container-platform.adoc [hunk 1]
```diff
@@ -6,6 +6,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+[NOTE]
```
Comment (kalexand-rh) [inline cli_reference/helm-cli/getting-started-with-helm-on-openshift-container-platform.adoc]: Please use the module instead: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features

Comment (boczkowska) [reply cli_reference/helm-cli/getting-started-with-helm-on-openshift-container-platform.adoc]: Thank you for pointing that out! Missed it somehow. Changed.

Comment (boczkowska) [reply cli_reference/helm-cli/getting-started-with-helm-on-openshift-container-platform.adoc]: Thank you for pointing that out! Missed it somehow. Changed.

**Comment by ahardin-rh** [comment]
LGTM! Thanks!

**Pull request merged**